setup.py

```diff
@@ -13,5 +13,5 @@
     package_data={'': ['templates/firebase_authentication/*.html']},
     include_package_data=True,
     python_requires='>=3.6',
-    install_requires=["firebase_admin~=5.2.0"],
+    install_requires=["firebase_admin~=6.5.0"],
```

Comment:
perhaps `>=6.5` would be better for future proofing this
```suggestion
    install_requires=["firebase_admin>=6.5.0"],
```

Reply:
I'm not sure if that's a good idea.
If a breaking change is introduced in the `firebase-admin`, dependent projects can end up in a malfunctioning state.
And it looks like it's not unusual for firebase to introduce such changes

Reply:
Applications should do the locking on their side, not libraries, otherwise you endup with having to do PR like this one here which is just a waste of time.

But since we are deprecating firebase and its not like this PR makes something worse, we can proceed with it as is.

Reply:
I generally agree, but the problem is that the dependent project I'm currently working on, does not have such locking, it uses plain `requirements.txt`. And I don't know how many other dependent projects do that

Reply:
fyi all new projects use pdm for locking and in the old ones, if needed, we can always use == in requirements.txt